### ISSUE TITLE
Is MapAnything depth map affine invariant?

### ISSUE THREAD

**Issue by mramezani64** [facebookresearch/map-anything#168]
Hi, 
Thank you for your valuable work. 
I just have a question. I wonder if we can assume MA's depth map is affine invariant?
Thanks

**Comment by Nik-V9** [comment]
Hi, thanks for trying out the work!

No, MA's depth maps are not affine invariant. The model predicts depth maps in a metric scale determined by the model (best guess), where within a scene the scale estimate should be conistent.

**Comment by mramezani64** [comment]
@Nik-V9  Thank you for your prompt reply. In the example below, I ran MA twice using four views from a similar scene. I fed camera poses from stereo visual odometry, so I assumed `is_metric_scale = true` . However, as seen, the predicted metric scale does not align well with the prior LiDAR map (green points) and comparison between the two MA outputs shown in blue and orange. I am not guiding depth using stereo depth as it violates the internal map consistency. This observation makes me feel depth maps are affine invariant although MA predicts a metric scale. Any insights would be appreciated. 
<img width="1153" height="679" alt="Image" src="https://github.com/user-attachments/assets/b1295381-48df-4d83-88f9-85844d472a48" />

**Comment by Nik-V9** [comment]
Just so I understand better, what is the difference in MA inference between the orange and blue pointclouds - one is with poses for stereo VO and other is without?

**Comment by Nik-V9** [comment]
Could you also share how exactly you are calling .infer()?

**Comment by mramezani64** [comment]
Both PCs are with SVO poses fed into MA. Nothing has changed in terms of input except slightly different views. I am calling .infer() by following demo files. The snippet is

```                
              with torch.no_grad():
                    preds = model.infer(
                        processed_views,
                        memory_efficient_inference=True,
                        use_amp=True,
                        use_multiview_confidence=True,
                        apply_mask=True,
                        apply_confidence_mask=True,
                        confidence_percentile=getattr(ma_params, 'confidence_percentile', 10.0), #configurable
                        ignore_calibration_inputs=False,
                        ignore_depth_inputs=True,
                        ignore_pose_inputs=False,
                        ignore_pose_scale_inputs=False, 
                    )
```
